deployment/deploy-data

```diff
@@ -6,72 +6,90 @@ set -o errexit
 
 # 1. Environment (beta or production)
 
-# 2. A path to a release directory of the format release-{mm-dd-yy} containing
-#    built_with_change_types.tsv, removed.tsv, version.json, diff.json, reports.tsv,
-#    and release-{mm-dd-yy}.tar.gz.
+# 2. A path to a release archive of the format release-{mm-dd-yy}.tar.gz
 
 # Example:
 
-# Running `./deployment/deploy-data beta PATH/TO/release-10-06-16`
+# Running `./deployment/deploy-data beta PATH/TO/release-10-06-16.tar.gz`
 # would add the 10-06-16 release to the beta database and put files in the correct
 # locations for access, storage and download.
 
+function add_release_tar_to_db {
+    STAGING=$1
+    TAR=$2
+    MANAGE_SCRIPT=$3
+
+    RELEASE_DIR="${STAGING}/output/release"
+
+    DATA=${RELEASE_DIR}/built_with_change_types.tsv
+    REMOVED=${RELEASE_DIR}/diff/removed.tsv
+    METADATA=${RELEASE_DIR}/metadata/version.json
+    DIFFJSON=${RELEASE_DIR}/diff/diff.json
+    REPORTS=${RELEASE_DIR}/artifacts/reports.tsv
+
+    # unpacking of tar into a staging dir
+    [ -e ${STAGING} ] && rm -r ${STAGING}
+    mkdir -p ${STAGING}
+    tar zxf ${TAR} -C ${STAGING}
+
+    echo "Adding release to database"
+
+    # add data to DB
+    python ${MANAGE_SCRIPT} addrelease ${DATA} ${REPORTS} ${METADATA} ${REMOVED} ${DIFFJSON}
+}
+
+
 HOST=${HOST:-brcaexchange.cloudapp.net}
 USER=brca
 
-
 # Expects environment (local,beta or production) as first argument.
 ENVIRONMENT=$1
 if [ "${ENVIRONMENT}" != "beta" ] && [ "${ENVIRONMENT}" != "production" ] && [ "${ENVIRONMENT}" != "local" ]  ;then
     echo "ERROR: Please only enter 'local', 'beta' or 'production' as environment."
     exit 1
 fi
 
-# Expects path to release directory as second argument. Removes trailing / if present.
-RELEASEDIRECTORYPATH=$2
-if [ "${RELEASEDIRECTORYPATH: -1}" == "/" ] ;then
-    RELEASEDIRECTORYPATH=${RELEASEDIRECTORYPATH%?}
-fi
-
-RELEASEDIRECTORY=$(basename $RELEASEDIRECTORYPATH)
+# Expects path to release archive as second argument. Removes trailing / if present.
```

Comment:
this comment should be updated -- `Removes trailing / if present.` is no longer true